examples/tokenvm/DEVNETS.md

```diff
@@ -80,12 +81,12 @@ Now we can spin up a new network of 6 nodes with some defaults:
 ```bash
 /tmp/avalancheup-aws default-spec \
 --arch-type amd64 \
---rust-os-type ubuntu20.04 \
+--os-type ubuntu20.04 \
 --anchor-nodes 3 \
 --non-anchor-nodes 3 \
 --regions us-west-2 \
 --instance-mode=on-demand \
---instance-types=c5.4xlarge \
+--instance-types='{"us-west-2":["c5.4xlarge"]}' \
```

Comment:
A lot of these changed in other branches, thanks for reminding me this was broken.